tests/test_condate.py

```diff
@@ -0,0 +1,75 @@
+from datetime import date, datetime
```

Comment:
can you move this into `test_datetime_parse.py`, We could do with rearranging tests, putting some tests in this file will cause more confusion.